### DISCUSSION TITLE
❓ Template deprecation

### DISCUSSION THREAD

**Discussion by kiranbayram** [tenpureto/tenpureto#140] [Q&A]
Is there a way to define a template as deprecated and add some kind of message to the newer version?

**Comment by rtimush** [comment]
@kiranbayram Do you mean deprecating the whole template repository or deprecating some branches?

**Reply by kiranbayram** [reply]
the whole repository

**Reply by rtimush** [reply]
No, there is no such functionality. I guess as a workaround you can mark all branches as `stability: deprecated` and add a message to branch descriptions.

**Reply by kiranbayram** [reply]
Okay, thanks for the information 👍
